### ISSUE TITLE
Please improve documentation

### ISSUE THREAD

**Issue by mrsalehi** [mlcommons/croissant#694]
Hi,

I was wondering if you could improve the documentation of the tool and include more examples on how to export different file formats (json, jsonl, etc.) to the croissant format. The current instructions are not clear. For instance, there are examples in the README that shows how to consume a croissant file or what it looks like but no examples on how to create it. Also could you please clarify what different field types are?

Thanks.

**Comment by XenonLamb** [comment]
I agree. The documentation AND the ipynb examples are too vague to follow. What makes it more confusing to me is that the https://github.com/mlcommons/croissant/blob/main/python/mlcroissant/recipes/introduction.ipynb example can't run by itself.

**Comment by luisoala** [comment]
#695
